Examples/Example-iOS/Source/GTMAppAuthExampleViewController.m

```diff
@@ -39,7 +39,7 @@
         https://console.developers.google.com/apis/credentials?project=_
         The client should be registered with the "iOS" type.
  */
-static NSString *const kClientID = @"YOUR_CLIENT.apps.googleusercontent.com";
+static NSString *const kClientID = @"687389107077-8qr6dh8fr4uaja89sdr5ieqb7mep04qv.apps.googleusercontent.com";
```

Comment:
Is this a test project of yours?

Reply:
It is. I will remove before review (converted this PR back to draft).